src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -1,29 +1,66 @@
 import styled from "@emotion/styled";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Link } from "react-router-dom";
+import { TaskStatusIcon } from "components/TaskStatusIcon";
 import { taskStatusToCopy, mapUmbrellaStatusColors } from "constants/task";
+import { TaskStatus } from "types/task";
+import { pluralize } from "utils/string";
 
 interface Props {
-  status: string;
+  status: TaskStatus;
   count: number;
   onClick?: () => void;
+  statusCounts?: { [key: string]: number };
+  href: string;
 }
+
 export const GroupedTaskStatusBadge: React.FC<Props> = ({
```

Comment:
I think this component would benefit from some unit tests. To ensure the correct formatting and linkification is being applied.